packages/api/prisma/migrations/20220914164547_make_text_in_comment_revision_nullable/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "comments.revisions" ALTER COLUMN "text" DROP NOT NULL;
```

Comment:
Needed to save a revision, where user only wants to save a title, but no text

Reply:
What do you think of copying over the text? Would make things easier maybe down the line

Reply:
How do you mean exactly by "copying over the text"? Copy into an existing migration file?

Reply:
When you create a new revision you copy over all the new data and only update what you send to update.

Reply:
yes. that is exactly the use case.